phpBB/adm/style/admin.css

```diff
@@ -2698,13 +2699,15 @@ fieldset.permissions .padding {
 }
 
 .send-stats-settings input[type=checkbox] + label:before {
-	content: "\f096";
-	font-family: FontAwesome;
+	content: "\f0c8";
```

Comment:
This wil give issues in master branch.
There will be merged conflicts during merging anyway where we can resolve it, but perhaps it's better to do it here already. `font-size` should come before `font-weight`, and `content` should come after all `font-` properties. 
And the font family will need double quotes, rather than single quotes:
```css
.send-stats-settings input[type="checkbox"] + label:before {
	font-family: "Font Awesome 5 Free";
	font-size: 1.5em;
	font-weight: bold;
	content: "\f096";
}
```